documentation/_components/activity_log.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: component
+type: component
+title: Activity log
+---
+
+<div class="activity-log">
```

Comment:
`<ul>` sounds fine for me. I've found that we don't have to be as prescriptive on cg-style as wds.